### DISCUSSION TITLE
How to change the log level when using docker deployment？

### DISCUSSION THREAD

**Discussion by zfanswer** [RamiAwar/dataline#247] [General]
I would like to confirm the whole process details includes openai api calls&response in docker log.
is there any way to do this?

**Comment by anthony2261** [comment]
Hey @zfanswer, can you tell us a bit more about your use case? Dataline currently provides optional logging with [LangSmith](https://www.langchain.com/langsmith) to log the OpenAI API calls and responses, but we know that that might not be feasible for everyone, especially when data privacy is necessary.

**Reply by RamiAwar** [reply]
@zfanswer what kind of logs do you want to see? Less or more? 😄 Not clear to me

**Comment by anthony2261** [comment]
Closing until we get more context 🤗
